Modules/Microsoft365DSC/DSCResources/MSFT_TeamsGroupPolicyAssignment/MSFT_TeamsGroupPolicyAssignment.psm1

```diff
@@ -0,0 +1,439 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Id,
```

Comment:
ID shouldn't be mandatory since it is not something returned by the Get method. Instead, we should look at making the GroupDisplayName/PolicyName pair mandatory to uniquely identify an instance.

Reply:
Should we even keep ID as a parameter here since it doesn't exist on the "server"? I would recommend simply dropping it at this stage.

Reply:
Maybe a better Key is Priority since it is unique?